### ISSUE TITLE
Use configuration file for libnaw

### ISSUE THREAD

**Issue by deavmi** [nawproject/libnaw#1]
This will allow BD_ERR to be turned off. Idk bruh.

**Comment by deavmi** [comment]
How should BD_ERR be turned off, I mam thinking a config file @Walkman100

**Comment by Walkman100** [comment]
isn't there already a config file, for saving homepage and whether to open it on startup or not

**Comment by deavmi** [comment]
No. For naw there is. Not for libnaw.

On 18 April 2016 11:32:49 PM SAST, "░̟▒̱̈́͑ͩ▓̰̣̹̝ͪ̀̔̈͠█͒ͨ��│̺̫͚͛̐̎́͊【W̳̬̽́̆̌ȁ̧̩̥͇̄͒l͇̆̎̎̐ͥk͚̙͚͇̈́̈ͣ̍̽̓m̜̤̰͚̼̟a̝̘͎̍̎n̛̝̤͗̅̈̇͗ͨ̾ͅ】͚̝͚̤̗̮ͩ̉̏͢│̧̲͚̺͎̪̾ͮ█̈́��̞̱̙̭͘▓̪̙̝̜̹̬▒̶̄░̵̭̫̤��" <notifications@github.com> wrote:
>isn't there already a config file, for saving homepage and whether to
>open it on startup or not
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nawproject/libnaw/issues/1#issuecomment-211592261

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Walkman100** [comment]
Ah. And is it a bad idea to just use the naw config file?

**Comment by deavmi** [comment]
Yeah. Because libnaw is a library and must be able to be used with any script besides the Naw.sh script.

On 19 April 2016 6:54:35 AM SAST, "░̟▒̱̈́͑ͩ▓̰̣̹̝ͪ̀̔̈͠█͒ͨ��│̺̫͚͛̐̎́͊【W̳̬̽́̆̌ȁ̧̩̥͇̄͒l͇̆̎̎̐ͥk͚̙͚͇̈́̈ͣ̍̽̓m̜̤̰͚̼̟a̝̘͎̍̎n̛̝̤͗̅̈̇͗ͨ̾ͅ】͚̝͚̤̗̮ͩ̉̏͢│̧̲͚̺͎̪̾ͮ█̈́��̞̱̙̭͘▓̪̙̝̜̹̬▒̶̄░̵̭̫̤��" <notifications@github.com> wrote:
>Ah. And is it a bad idea to just use the naw config file? 
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nawproject/libnaw/issues/1#issuecomment-211728648

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deavmi** [comment]
I will probably give it a separate config. Idk. Maybe BD_ERR should be set to false for now. Idk.

On 19 April 2016 6:54:35 AM SAST, "░̟▒̱̈́͑ͩ▓̰̣̹̝ͪ̀̔̈͠█͒ͨ��│̺̫͚͛̐̎́͊【W̳̬̽́̆̌ȁ̧̩̥͇̄͒l͇̆̎̎̐ͥk͚̙͚͇̈́̈ͣ̍̽̓m̜̤̰͚̼̟a̝̘͎̍̎n̛̝̤͗̅̈̇͗ͨ̾ͅ】͚̝͚̤̗̮ͩ̉̏͢│̧̲͚̺͎̪̾ͮ█̈́��̞̱̙̭͘▓̪̙̝̜̹̬▒̶̄░̵̭̫̤��" <notifications@github.com> wrote:
>Ah. And is it a bad idea to just use the naw config file? 
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/nawproject/libnaw/issues/1#issuecomment-211728648

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Walkman100** [comment]
Ok, so then just add another config file next to the naw one.

**Comment by deavmi** [comment]
@Walkman100 Yep. Probably will do that.

**Comment by deavmi** [comment]
Will add config file soon. Just thinking about it I guess.
